Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -117,11 +166,15 @@ public static MixedRealityManager Instance
                 {
                     return instance;
                 }
-
+//Modified to enable Unit Tests
+#if UNITY_EDITOR
+                if (UnityEditor.EditorApplication.isPlaying && !searchForInstance)
```

Comment:
Let's replace with `Application.IsEditor` so we can get rid of the `#Ifdef`'s

Reply:
I forgot about that one, thanks